src/Symfony/Component/Finder/Tests/FinderTest.php

```diff
@@ -26,55 +26,80 @@ public function testDirectories()
     {
         $finder = $this->buildFinder();
         $this->assertSame($finder, $finder->directories());
-        $this->assertIterator($this->toAbsolute(array('foo', 'toto')), $finder->in(self::$tmpDir)->getIterator());
+        $this->assertIterator(self::toAbsolute(array('foo', 'toto')), $finder->in(self::$tmpDir)->getIterator());
```

Comment:
please revert all these changes, they are unrelated to the PR and make reviewing harder than strictly required

Reply:
yeah, sorry. Should be fixed, but of course is unrelated to this change